lib/raven/instance.rb

```diff
@@ -0,0 +1,219 @@
+require 'raven/base'
+require 'English'
```

Comment:
This was from rubocop - which request that `$!` on L211 be replaced with $ERROR_INFO from this module.